DrinkACupOfJuice/Q&A2.0/answer.js

```
@@ -0,0 +1,105 @@
+                if (!response.ok) {
+                    throw new Error('Network response was not ok');
+                }
+                return response.json();
+            })
+            .then(data => {
+                const container = document.getElementById('questionsAndAnswers');
+                container.innerHTML = ''; // 清空容器内容
+
+                if (Array.isArray(data)) {
+                    data.forEach(questionObj => {
+                        if (questionObj.hasOwnProperty('id') && questionObj.hasOwnProperty('title') && questionObj.hasOwnProperty('answers')) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)